db/patterns/blockthrough.eno

```diff
@@ -17,5 +17,3 @@ pagefair.net
 ||pagefair.net^$3p
 ||s3.amazonaws.com/asset.pagefair.com
 --- filters
```

Comment:
```suggestion
--- filters

ghostery_id: 1628
```

For new entries, it is indeed recommended to omit the `ghostery_id`. For historic entries that already have them, it is safer to leave them in for now.